server/manager/src/main/java/org/apache/accumulo/manager/ManagerClientServiceHandler.java

```diff
@@ -591,15 +584,6 @@ private void alterTableProperty(TCredentials c, String tableName, String propert
     }
   }
 
-  private void updatePlugins(String property) {
-    // resolve without warning; any warnings should have already occurred
-    String resolved = DeprecatedPropertyUtil.getReplacementName(property, (log, replacement) -> {});
-    if (resolved.equals(Property.MANAGER_TABLET_BALANCER.getKey())) {
-      manager.initializeBalancer();
-      log.info("tablet balancer changed to {}", manager.getBalancerClass().getName());
```

Comment:
Is the getBalancerClass() method in the manager still used by other stuff?  Wondering if it could be removed.

Reply:
It was not being used anymore, removed it in fcd25da.